addons/web_editor/static/lib/summernote/src/js/Renderer.js

```diff
@@ -462,6 +462,21 @@ define([
           event: 'imageShape',
           value: 'img-thumbnail'
         });
+        var leafButton = tplIconButton(options.iconPrefix + options.icons.image.shapeLeaf, {
+          title: lang.image.shapeLeaf,
+          event: 'imageShape',
+          value: 'rounded-leaf'
+        });
+        var pillButton = tplIconButton(options.iconPrefix + options.icons.image.shapePill, {
+          title: lang.image.shapeLeaf,
```

Comment:
@Cocographique I'd prefer not to modify the editor bar right now. Also here you use the wrong variable. Could you take this commit out in a new PR? Again: no need to remove it from this one, I will push forced anyway.

Reply:
[**PR 46848**](https://github.com/odoo/odoo/pull/46848)

Reply:
**PR** [46848](https://github.com/odoo/odoo/pull/46848)